x-pack/plugin/sql/sql-action/src/test/java/org/elasticsearch/xpack/sql/action/SqlRequestParsersTests.java

```diff
@@ -66,10 +68,11 @@ public void testClearCursorRequestParser() throws IOException {
         assertNull(request.clientId());
         assertEquals(Mode.PLAIN, request.mode());
         assertEquals("whatever", request.getCursor());
-        
-        request = generateRequest("{\"cursor\" : \"whatever\", \"client_id\" : \"CLI\"}",
+
+        request = generateRequest("{\"cursor\" : \"whatever\", \"client_id\" : \"CLI\", \"client_version\": \"1.2.3\"}",
                 SqlClearCursorRequest::fromXContent);
         assertNull(request.clientId());
+        assertNull(request.version());
```

Comment:
Why is it null here?

Reply:
b/c the parser for `SqlClearCursorRequest` doesn't set these fields.